spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -3,39 +3,46 @@
 describe DispatchRider::Subscriber do
 
   before do
-    DispatchRider::Handlers::Base.stub(:subclasses){ Set.new }
+    allow(DispatchRider::Handlers::Base).to receive(:subclasses) { Set.new }
 
-    stub_const("FooBar", Class.new(DispatchRider::Handlers::Base) {
-      def process(options)
+    konst = Class.new(DispatchRider::Handlers::Base) do
+      def process(_options)
         throw :process_was_called
       end
-    })
+    end
+    stub_const("FooBar", konst)
   end
 
   describe "#initialize" do
     it "should assign a new queue service registrar" do
-      subject.queue_service_registrar.store.should be_empty
+      expect(subject.queue_service_registrar.store).to be_empty
     end
   end
 
   describe "#register_queue" do
     it "should register a queue service with the queue service registrar" do
       subject.register_queue(:simple)
-      subject.queue_service_registrar.fetch(:simple).should be_empty
+      expect(subject.queue_service_registrar.fetch(:simple)).to be_empty
     end
   end
 
   describe "#register_handler" do
     it "should register a handler" do
       subject.register_handler(:foo_bar)
-      expect { subject.dispatcher.dispatch(DispatchRider::Message.new(:subject => :foo_bar, :body => {'foo' => 'bar'})) }.to throw_symbol(:process_was_called)
+      expect {
+        message = DispatchRider::Message.new(:subject => :foo_bar, :body => {'foo' => 'bar'})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.